{
  "result": "success",
  "message": "[probe] After sched_setscheduler\n[probe] sched_setscheduler returned: ret=-1, errno=1\n[probe] strerror(ret) -> Unknown error -1\n[probe] strerror(errno) -> Operation not permitted\n[probe] After sched_setscheduler\n[probe] sched_setscheduler returned: ret=-1, errno=1\n[probe] strerror(ret) -> Unknown error -1\n[probe] strerror(errno) -> Operation not permitted\n"
}